ui/apps/platform/src/Containers/ComplianceEnhanced/ScanConfigs/Wizard/ReviewConfig.tsx

```diff
@@ -1,9 +1,93 @@
-import React, { ReactElement } from 'react';
+import React from 'react';
+import { FormikContextType, useFormikContext } from 'formik';
+import {
+    Divider,
+    Flex,
+    FlexItem,
+    PageSection,
+    Stack,
+    StackItem,
+    Text,
+    TextList,
+    TextListItem,
+    TextVariants,
+    Title,
+} from '@patternfly/react-core';
+
+import { ClusterScopeObject } from 'services/RolesService';
+import { ComplianceProfile } from 'services/ComplianceEnhancedService';
+
+import { ScanConfigFormValues } from './useFormikScanConfig';
+import {
+    convertFormikParametersToSchedule,
+    formatScanSchedule,
+} from '../compliance.scanConfigs.utils';
+
+export type ProfileSelectionProps = {
+    clusters: ClusterScopeObject[];
+    profiles: ComplianceProfile[];
+};
+
+function ReviewConfig({ clusters, profiles }: ProfileSelectionProps) {
+    const { values: formikValues }: FormikContextType<ScanConfigFormValues> = useFormikContext();
+
+    const scanSchedule = convertFormikParametersToSchedule(formikValues.parameters);
+    const formattedScanSchedule = formatScanSchedule(scanSchedule);
+
+    function findById<T extends { id: string }>(selectedIds: string[], items: T[]): T[] {
+        return selectedIds
+            .map((id) => items.find((item) => item.id === id))
+            .filter((item): item is T => item !== undefined);
+    }
+
+    const selectedClusters = findById<ClusterScopeObject>(formikValues.clusters, clusters);
+    const selectedProfiles = findById<ComplianceProfile>(formikValues.profiles, profiles);
 
-function ReviewConfig(): ReactElement {
     return (
         <>
-            <div>Review</div>
+            <PageSection variant="light" padding={{ default: 'noPadding' }}>
+                <Flex direction={{ default: 'column' }} className="pf-u-py-lg pf-u-px-lg">
+                    <FlexItem>
+                        <Title headingLevel="h2">Review</Title>
+                    </FlexItem>
+                    <FlexItem>Review and create your scan configuration</FlexItem>
+                </Flex>
+            </PageSection>
+            <Divider component="div" />
+            <Stack hasGutter className="pf-u-py-lg pf-u-px-lg">
```

Comment:
For information, not a change request: PatternFly `DescriptionList` is an alternative for future work:
https://www.patternfly.org/components/description-list